src/lib/serviceRoutinesV2/postBatchQuery.cpp

```diff
@@ -62,12 +65,42 @@ std::string postBatchQuery
 {
   BatchQuery*           bqP  = &parseDataP->bq.res;
   QueryContextRequest*  qcrP = &parseDataP->qcr.res;
+  std::string           q    = ciP->uriParam[URI_PARAM_Q];
   Entities              entities;
   std::string           answer;
 
   qcrP->fill(bqP);
   bqP->release();  // qcrP just 'took over' the data from bqP, bqP no longer needed
 
+  //
```

Comment:
Batch query (i.e. "POST /v2/op/query") doesn't allow `q` param. The parameters supported by this operation (see http://telefonicaid.github.io/fiware-orion/api/v2/) are limit, offset, orderBy and options.